src/manifold/src/manifold.cpp

```diff
@@ -904,6 +906,7 @@ CrossSection Manifold::Project() const {
 
 ExecutionParams& ManifoldParams() { return manifoldParams; }
 
+#ifdef MANIFOLD_HULL
```

Comment:
Like this?

Reply:
Yes. Do note that some of the tests will not compile now due to linker error, so we need to ifdef them as well.

Reply:
Already done.

Reply:
Hmm, or not. I got the tests, but not the bindings. I kind of feel like Hull shouldn't be optional for the bindings. Any thoughts for how I should set up the CI to test this?

Reply:
The simplest way is to put ifdef in `manifoldc.h`, like https://github.com/elalish/manifold/blob/6b4a4ed841357f627573181402d3bb979092b5bb/bindings/c/include/manifoldc.h#L414

Reply:
Wouldn't we then need to do the same for the Python and WASM bindings? This is why I was thinking of not letting these dependencies be optional for the bindings, since we package them all up and distribute them anyway.

Reply:
I think it may make sense for c binding, as there is no package manager for it and people need to compile on their own?

For python and js, I agree we can just not let them be optional.

Reply:
How would Python and JS customers know whether the hull feature is available?

Reply:
>For python and js, I agree we can just not let them be optional.

I think the optionality will only be for C++ consumers.